components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -43,6 +43,18 @@ module crm_physics
    ! Physics buffer indices  
    integer :: ttend_dp_idx    = -1
 
+   integer :: crm_t_rad_idx    = -1
+   integer :: crm_qv_rad_idx   = -1
+   integer :: crm_qc_rad_idx   = -1
+   integer :: crm_qi_rad_idx   = -1
+   integer :: crm_cld_rad_idx  = -1
+   integer :: crm_qrad_idx     = -1
+
+   integer :: crm_rad_idx_nc   = -1
```

Comment:
These should be named consistent with the other indices (e.g., `crm_nc_rad_idx`).

Reply:
shoot, totally missed those

Reply:
fixed.